### PR TITLE
[TC-9617] Rebrand to "single delivery"

### PR DESCRIPTION
**Pull request by marcmatt**
## Description

https://tradecloud.atlassian.net/browse/TC-9617

https://app.gitbook.com/o/-LNyIUZSvpZWZ81yEr7V/s/-LNyJ9UuwLCjTvA2sqQR-887967055/~/revisions/2b1C2rybUsksVl4uYRuy/

### Scope
This PR rebrands "simple delivery schedule" to "single delivery"

### Submitter pre-flight check

- [x] Review your documentation
- [x] Add labels `needs reviewing`
- [x] Assign PR and JIRA ticket to 'Reviewer' (in 'Reviewing' state)

### Reviewer pre-flight check

- [ ] Review documentation in PR
- [ ] Remove label `needs reviewing`
- [ ] Merge the PR and remove the release for this branch from Gitbook
- [ ] Set Jira ticket to 'Released'

### CODE REVIEW

File: order/buyer/cancel.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Cancel an order line in Tradecloud when it is cancelled at the buyer.
 * `Cancelled` lines cannot be cancelled again
 
 {% hint style="warning" %}
-When using the simple delivery schedule, the `cancelled` indicator is only supported for the first order line of each item number. The other lines with the same item number will also be cancelled together with the first line.
+When using the single delivery per order line feature, the `cancelled` indicator is only supported for the first order line of each item number. The other lines with the same item number will also be cancelled together with the first line.
```
Comment (Sjorstradecloud) [inline order/buyer/cancel.md:14]: Good we mention this.

File: order/buyer/issue/delivery-schedule.md [hunk 2]
```diff
@@ -1,31 +1,30 @@
 ---
 description: >-
-  Choose between the native or simple delivery schedule
+  Choose between delivery schedule or single delivery per order line
```
Comment (Sjorstradecloud) [inline order/buyer/issue/delivery-schedule.md:3]: Might be handy to make this page a 1 on 1 copy of https://app.gitbook.com/o/-LNyIUZSvpZWZ81yEr7V/s/-LNyJ9UuwLCjTvA2sqQR-887967055/~/revisions/2b1C2rybUsksVl4uYRuy/introduction/api/delivery-schedule

**Review by Sjorstradecloud — APPROVED** [review]

File: order/buyer/issue/delivery-schedule.md [hunk 3]
```diff
@@ -1,31 +1,30 @@
...
 ---
 
 # Send a delivery schedule
 
-## Native versus simple delivery schedule
+## Ddelivery schedule versus single delivery
```
Comment (roy-tc) [inline order/buyer/issue/delivery-schedule.md:8]:
```suggestion
## Delivery schedule versus single delivery
```

File: order/supplier/receive/single-delivery-order-event.md [hunk 4]
```diff
@@ -1,18 +1,18 @@
 ---
-description: How to receive a simple order sent by the buyer.
+description: How to receive a single delivery order sent by the buyer.
 ---
 
-# Simple Order Event
+# Som Order Event
```
Comment (roy-tc) [inline order/supplier/receive/single-delivery-order-event.md:5]: Typo

File: order/buyer/receive/single-delivery-order-event.md [hunk 5]
```diff
@@ -6,13 +6,13 @@ description: How to receive a simple order response sent by the supplier
 
 Tradecloud will send a purchase order response to the buyer when an order event has been triggered.
```
Comment (roy-tc) [inline order/buyer/receive/single-delivery-order-event.md:7]: Note: the page description still says:

```
description: How to receive a simple order response sent by the supplier
```

Also reword "simple" to "single" here.

**Review by roy-tc — CHANGES_REQUESTED** [review]
Very nice! It sounds a lot more intuitive now.

There are some small typo's that have to be fixed, but otherwise looks good to go.

File: order/buyer/issue/delivery-schedule.md (hunk 2 above)
Comment (marcmatt) [reply order/buyer/issue/delivery-schedule.md:3]: You are right, there is a great overlap, the general one, is general, and this one is focused on issuing.
The delivery schedule vs single delivery be rewritten as part as the new API guide.

File: order/supplier/receive/single-delivery-order-event.md (hunk 4 above)
Comment (marcmatt) [reply order/supplier/receive/single-delivery-order-event.md]: fixed

File: order/buyer/receive/single-delivery-order-event.md (hunk 5 above)
Comment (marcmatt) [reply order/buyer/receive/single-delivery-order-event.md:7]: fixed

**Pull request merged**